CONTRIBUTING.md

```diff
@@ -80,34 +93,21 @@ You can find the language's `code, name, localeName` in this link
 https://github.com/ethereum/ethereum-org-website/blob/dev/i18n.config.json
 
 ### Whether or not to use `defaultMessage`?
-If you search `FormattedMessage` or `intl.formatMessage` in this project, you will notice that most of them only have a `id` prop, but a few of them have a `defaultMessage` prop.
+If you search `FormattedMessage` or `intl.formatMessage` in this project, you will notice that most only have a `id` prop, but a few of them have a `defaultMessage` prop.
 
 **Why?**
 
-Each non-english language will be filled in the gaps with english. Even though there may be some un-translated content, it will always use english as defaultMessage. That's why we don't need to provide a `defaultMessage` prop each time we render a `FormattedMessage` component.
+The gaps in an incomplete non-English language will be filled with English. The un-translated content will use English as defaultMessage. That's why we don't need to provide a `defaultMessage` prop each time we render a `FormattedMessage` component.
 
 But in some cases, the `id` prop may not be static. For example,
 ```jsx
 <h6 className="pt-0 mb-1" data-id='sidePanelSwapitTitle'>
  <FormattedMessage id={plugin?.profile.name + '.displayName'} defaultMessage={plugin?.profile.displayName || plugin?.profile.name} />
 </h6>
 ```
-You can't be sure whether there is a match key in locale file or not. So it will be better to provide a `defaultMessage` prop.
-
-### Should I update the non-english locale json files?
-You probably will have this question when you are updating the english locale json files. 
+You can't be sure whether there is a match key in locale file or not. So its better to provide a `defaultMessage` prop.
```

Comment:
This might read better this way: "You can't be sure whether there is a matched key in the locale file or not..."